src/components/MagicForm/MagicForm.spec.js

```diff
@@ -4,37 +4,30 @@ import userEvent from '@testing-library/user-event';
 import MagicForm from './MagicForm.js';
 
 describe('MagicForm', () => {
-  it('is an accessible form', () => {
-    render(<MagicForm />);
+  it('Form is an accessible form', () => {
+    render(<MagicForm savedOrders={[]} />);
     const form = screen.getByRole('form');
     expect(form).toBeInTheDocument();
   });
-  it('Button of Form is an accessible button', () => {
-    render(<MagicForm />);
-    const button = screen.getByRole('button');
-    expect(button).toBeInTheDocument();
-  });
+
   it('Input of Form is an accessible input', () => {
-    render(<MagicForm />);
+    render(<MagicForm savedOrders={[]} />);
     const input = screen.getByLabelText('type in your command');
     expect(input).toBeInTheDocument();
   });
 
-  it('MagicForm onSubmit function gets called with Enter on keyboard', async () => {
-    const user = userEvent.setup();
-    const callback = jest.fn();
-    render(<MagicForm analyzeSpell={callback} />);
-    const input = screen.getByLabelText('type in your command');
-    await user.type(input, 'something should happen{Enter}');
-    expect(callback).toHaveBeenCalled();
+  it('MagicForm has 2 buttons and both are accessible', async () => {
+    render(<MagicForm savedOrders={[]} />);
+    const button = screen.getByRole('button');
```

Comment:
I think getAllByRole is the better choice here. 
`expect(buttons.lenght).toBe(2)`